lib/main_common.pm

```diff
@@ -2796,6 +2796,13 @@ sub load_publiccloud_tests {
         loadtest "publiccloud/img_proof";
     }
     elsif (get_var('PUBLIC_CLOUD_QAM')) {
+        if (get_var('PUBLIC_CLOUD_QAM_IMG_PROOF_TESTS')) {
+            # setting 'PUBLIC_CLOUD_IMG_PROOF_TESTS' is used by img_proof
+            set_var('PUBLIC_CLOUD_IMG_PROOF_TESTS', get_var('PUBLIC_CLOUD_QAM_IMG_PROOF_TESTS'));
```

Comment:
why we need QAM flavor of the same variable ?

Reply:
Because if we set `PUBLIC_CLOUD_IMG_PROOF_TESTS` then the QAM tests don't get scheduled:

        elsif (get_var('PUBLIC_CLOUD_IMG_PROOF_TESTS')) {
            loadtest "publiccloud/img_proof";
        }
        elsif (get_var('PUBLIC_CLOUD_QAM')) {

This might be something that we want to change in `main_common.pm` - for now I'm playing around with another variable.

Reply:
I changed it now accordingly. It does indeed not make any sense to have another variable name.